client/src/Login.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+
+
+const Login = () => {
+  const [username, setUsername] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [registerMode, setRegisterMode] = useState(false);
+
+  const toggleRegister = () => {
+    setRegisterMode(!registerMode);
+  };
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+
+    // user submitted data
+    const data = {
+      username,
+      email,
+      password
+    };
+
+    // make API call.
+    try {
+      let api = registerMode ? 'http://localhost:3000/api/auth/register' : 'http://localhost:3000/api/auth/login';
+      console.log(data);
+      console.log(`Mode is: ${api}`);
+        const response = await fetch(api, {
+          method: 'POST',
+          headers: {'Content-Type': 'application/json'},
+          body: JSON.stringify(data),
+      });
+
+      if(!response.ok) {
+        throw new Error(`Login request failed: ${response.status}`);
+      }
+
+      const responseData = await response.json();
+    } catch (error) {
+      // Handle error
+    }
+  };
+
+  console.log(registerMode);
+  return (
+    <div>
+      <div> { registerMode ?
+        (<div>
+          <h1>Register</h1>
+            <form onSubmit={handleSubmit}>
+            <div>
+              <label htmlFor="username" >Email</label>
```

Comment:
```suggestion
              <label htmlFor="username" >Username</label>
```

Reply:
Everything else looks great, but this one adjustment.

Reply:
Fixed in commit [7c9ca79](https://github.com/justinsprecco/Acumen/pull/4/commits/7c9ca79ec5db0a6c795fae729d5549b1d2e7ff16)